libcudacxx/include/cuda/std/__algorithm/copy.h

```diff
@@ -70,7 +70,7 @@ template <class _AlgPolicy,
           class _Tp,
           class _Up,
           __enable_if_t<_CCCL_TRAIT(is_same, __remove_const_t<_Tp>, _Up), int> = 0,
-          __enable_if_t<_CCCL_TRAIT(is_trivially_copy_assignable, _Up), int>   = 0>
+          __enable_if_t<_CCCL_TRAIT(is_trivially_copyable, _Up), int>          = 0>
```

Comment:
Those are actually bugfixes that we might want to pull out

Reply:
I think we might want to pull those out. It would help with the review. I also have an easier time approving a PR that only adds a new feature than one that changes existing functionality.

Reply:
Opened #1937